cypress-tests/cypress/support/commands.js

```diff
@@ -122,6 +122,12 @@ Cypress.Commands.add("ListConnectorsFeatureMatrixCall", (globalState) => {
     logRequestId(response.headers["x-request-id"]);
 
     expect(response.body).to.have.property("data").and.not.empty;
+    expect(response.body.data).to.be.an("array").and.not.empty;
```

Comment:
Does this work?

I guess it should be 
```suggestion
    expect(response.body.data).to.be.an("array").and.not.be.empty;
```

here and below...

Reply:
It works. The tests run successfully.
<img width="431" alt="Screenshot 2024-12-09 at 12 08 16 PM" src="https://github.com/user-attachments/assets/00125f17-5cd4-4c96-82b9-04e04c3b2498">